examples/linear_model/plot_poisson_regression_non_normal_loss.py

```diff
@@ -3,87 +3,86 @@
 Poisson regression and non-normal loss
 ======================================
 
-This example illustrates the use of log-linear Poisson regression
-on the `French Motor Third-Party Liability Claims dataset
-<https://www.openml.org/d/41214>`_ from [1]_ and compares
-it with models learned with least squared error. In this dataset, each sample
-corresponds to an insurance policy, i.e. a contract within an insurance
-company and an individual (policiholder). Available features include driver
-age, vehicle age, vehicle power, etc.
-
-A few definitions: a *claim* is the request made by a policyholder to the
-insurer to compensate for a loss covered by the insurance. The *exposure* is
-the duration of the insurance coverage of a given policy, in years.
-
-Our goal is to predict the expected number of insurance claims (or frequency)
-following car accidents for a policyholder given the historical data over a
-population of policyholders.
+This example illustrates the use of log-linear Poisson regression on the
+`French Motor Third-Party Liability Claims dataset
+<https://www.openml.org/d/41214>`_ from [1]_ and compares it with models
+learned with least squared error.
+
+A few definitions:
+
+- a **policy** is a contract between an insurance company and an individual:
+  the **policyholder**, that is, the vehicle driver in this case.
+
+- a **claim** is the request made by a policyholder to the insurer to
+  compensate for a loss covered by the insurance.
+
+- the **exposure** is the duration of the insurance coverage of a given policy,
+  in years.
+
+- the claim **frequency** is the number of claims divided by the exposure,
+  typically measured in number of claims per year.
+
+In this dataset, each sample corresponds to an insurance policy. Available
+features include driver age, vehicle age, vehicle power, etc.
+
+Our goal is to predict the expected frequency of claims following car accidents
+for a new policyholder given the historical data over a population of
+policyholders.
 
 .. [1]  A. Noll, R. Salzmann and M.V. Wuthrich, Case Study: French Motor
-    Third-Party Liability Claims (November 8, 2018).
-    `doi:10.2139/ssrn.3164764 <http://dx.doi.org/10.2139/ssrn.3164764>`_
+    Third-Party Liability Claims (November 8, 2018). `doi:10.2139/ssrn.3164764
+    <http://dx.doi.org/10.2139/ssrn.3164764>`_
 
 """
 print(__doc__)
-
 # Authors: Christian Lorentzen <lorentzen.ch@gmail.com>
 #          Roman Yurchak <rth.yurchak@gmail.com>
+#          Olivier Grisel <olivier.grisel@ensta.org>
 # License: BSD 3 clause
 import warnings
-
 import numpy as np
 import matplotlib.pyplot as plt
 import pandas as pd
 
-from sklearn.datasets import fetch_openml
```

Comment:
I really prefer to split large import blocks for long examples. Otherwise the reader as to scroll-up a lot to know where a function / class comes from.